### PR TITLE
Update `ContactPatientBottomSheet` Behaviour - 1

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.clubhouse.io/simpledotorg/story/3714/update-contactpatientbottomsheet-behaviour?vc_group_by=day&ct_workflow=all&cf_workflow=500000005

### CODE REVIEW

File: app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt [hunk 1]
```diff
@@ -2501,6 +2501,53 @@ class AppointmentRepositoryAndroidTest {
     assertThat(latest_appointment_a_week_later.get().appointment).isEqualTo(appointment_scheduled_for_today_with_reminder_a_week_in_the_future)
   }
 
+  @Suppress("LocalVariableName")
```
Comment (msasikanth) [inline app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt:2504]: What's this suppress annotation for? 🤔

**Review by msasikanth — COMMENTED** [review]
@janhavisinghh Correct me if I am wrong, the query we have already loaded patient data with and without phone numbers right? Is there a reason we need a new query?

We already do this filtering when loading overdue appointments based on the feature flag, so do we need to have a separate query here again?

**Comment by janhavisinghh** [comment]
@msasikanth we need this because we use a separate query to get overdue appointment of a patient while loading the bottomsheet. The databaseview has null phone numbers in it which is why we needed this change. In the last change we only made changes to the query which loads overdue appointments in the overdue list screen.

**Comment by msasikanth** [comment]
> 
> 
> @msasikanth we need this because we use a separate query to get overdue appointment of a patient while loading the bottomsheet. The databaseview has null phone numbers in it which is why we needed this change. In the last change we only made changes to the query which loads overdue appointments in the overdue list screen.

I mean we are already loading the overdue list without phone numbers right (by default, when feature flag is off)? What are the chances that the phone number will be null when we just get a single patient? If the phone number is null they won't be visible in the list in the first place right? We can get on a call if you want to discuss this.

**Comment by janhavisinghh** [comment]
Oooohhh. Fair enough. Got it. We don't need this change🤦🏻‍♀️. We can close this PR.

**Pull request closed without merging**